external-crates/move/crates/move-compiler/src/typing/syntax_methods.rs

```diff
@@ -173,7 +173,7 @@ fn validate_index_syntax_methods(
         index_fn,
         Some(mut_tparam_types),
     );
-    context.current_module = None;
+    // context.current_module = None;
```

Comment:
This was done to improve error reporting, but it appears that is no longer an issue?

Reply:
We don't have a ModuleContext here, so it is None